src/sonic-yang-models/yang-models/sonic-restapi.yang

```diff
@@ -0,0 +1,63 @@
+module sonic-restapi {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-restapi";
+    prefix restapi;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description "RESTAPI YANG Module for SONiC OS";
+
+    revision 2022-10-05 {
+        description "First Revision";
+    }
+
+    container sonic-restapi {
+
+        container RESTAPI {
+
+            description "RESTAPI TABLE part of config_db.json";
+
+            container certs {
+
+                leaf ca_crt {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-]+)*/([a-zA-Z0-9_-]+).pem';
+                    }
+                    description "Local path for ca_crt.";
+                }
+
+                leaf server_crt {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-]+)*/([a-zA-Z0-9_-]+).crt';
+                    }
```

Comment:
Do we really need to limit the suffix as crt? Can we use xxx.cer?

Reply:
i think for restapi, it is currently fixed to crt. We may not need to provide an extension support